MPU/src/cascadiamc.cpp

```diff
@@ -92,17 +92,6 @@ uint16_t CascadiaMC::getTorque()
 }
 
 
-void CascadiaMC::clearFault()
-{
-    int time = millis() + 250;
-    while(millis() < time) {
-        sendMessageCAN1(CANMSG_MC_SETPARAMETER, 8, FAULT_CLEAR);
-        delay(5);
-    }
-    isFaulted = false;
```

Comment:
This "isFaulted" variable is not addressed elsewhere?

Reply:
it is, but it will get passed when we clear the fault manually i believe

Reply:
Passed?

Reply:
like set to false

Reply:
so like this automatically did it, but when we press the button the clear fault thing will do it itself

Reply:
I'm worried we may be missing a step here, but we can diagnose tomorrow on the actual car